pkg/ddevapp/config.go

```diff
@@ -930,14 +930,14 @@ redirect_stderr=true
 		return "", err
 	}
 
-	// Add .pgpass to homedir on postgres
+	// Add .pgpass to homedir on PostgreSQL
 	extraDBContent := ""
 	if app.Database.Type == nodeps.Postgres {
-		// Postgres 9/10/11 upstream images are stretch-based, out of support from Debian.
-		// Postgres 9/10 are out of support by Postgres and no new images being pushed, see
+		// PostgreSQL 9/10/11 upstream images are stretch-based, out of support from Debian.
+		// PostgreSQL 9/10 are out of support by PostgreSQL and no new images being pushed, see
 		// https://github.com/docker-library/postgres/issues/1012
-		// However, they do have a postgres:11-bullseye, but we won't start using it yet
-		// because of awkward changes to $DBIMAGE. Postgres 11 will be EOL Nov 2023
+		// However, they do have a Postgres:11-bullseye, but we won't start using it yet
```

Comment:
```suggestion
		// However, they do have a postgres:11-bullseye, but we won't start using it yet
```

Reply:
uhhhhh what an oversight on my end, that slipped through with the back and forth. thank you for noticing both cases!